src/Controls/src/Xaml/XamlServiceProvider.cs

```diff
@@ -147,7 +148,13 @@ public SimpleValueTargetProvider(object[] objectAndParents, object targetPropert
 		{
 		}
 
+		[Obsolete("Use the other ctor")]
```

Comment:
Just try to add more details about the suggested  alternative:
`[Obsolete("This constructor is deprecated. Use the SimpleValueTargetProvider constructor with INameScope[] scopes and object rootObject instead.")]`

Reply:
this is only used by generated code, so the obsolete message should never show up